mars/tensor/merge/tests/test_merge_execute.py

```diff
@@ -222,6 +222,21 @@ def testBlockExecution(self):
                           [1., 1., 0., 0., 3.]])
         np.testing.assert_array_equal(r, expected)
 
+        # eye with different chunk sizes
+        a = eye(2, chunk_size=5) * 2
+        b = eye(3, chunk_size=7) * 3
```

Comment:
when `chunk_size` is greater than tensor shape, the tensor still has one chunk.

Try to increase the shape like this `a = eye(9, chunk_size=5) * 2`

Reply:
Good suggestion. Thanks.